docker-entrypoint.sh

```diff
@@ -29,16 +29,21 @@ if [ "$1" = 'postgres' ]; then
 		fi
 		
 		: ${POSTGRES_USER:=postgres}
-		if [ "$POSTGRES_USER" = 'postgres' ]; then
-			op='ALTER'
-		else
-			op='CREATE'
+		: ${POSTGRES_DB:=$POSTGRES_USER}
```

Comment:
I don't think that makes a lot of sense unless you're also going to drop the `postgres` user. Seems like more trouble that it's worth and also like the sort of thing that could break implicit assumptions. It definitely doesn't need to be muddled into this PR.